requirements_dev.txt

```diff
@@ -2,9 +2,12 @@ autoflake==1.3
 bumpversion==0.5.3
 coverage==4.5.1
 flake8==3.5.0
+ipdb==0.12
+parameterized==0.7.0
 pip==18.1
 pre-commit==1.17.0
-
+pydeps==1.7.3
+pyflakes<1.7.0,>=1.5.0
```

Comment:
Autoflake and isort should be only in `setup.py` to avoid duplicate. In first step, we should install project depedencies. In second step, you should install developer requirements.